_posts/jeffrey_burt/2017-01-13-d3-brush.md

```diff
@@ -39,7 +39,7 @@ if(majorVersion === '3') {
 #### Method 2
 Because the API has changed so much, in a lot of cases you can simply check for the existence of the methods you need.
 
-For example, if the v3 method you need is `d3.scale.linear` and the v4 one is `d3.scaleLinear` you could do
+For example, if the v3 method you need is `d3.scale.linear` and the v4 one is `d3.scaleLinear` you could do  
```

Comment:
Yeah, I'm wondering if the code blocks are breaking without a newline ( code blocks are working after headers, but not in the two places I had no new line. 

But I'm really shooting in the dark here, cause it all works locally :)